src/test/kotlin/com/terraformation/backend/accelerator/ModuleEventNotifierTest.kt

```diff
@@ -76,7 +76,7 @@ class ModuleEventNotifierTest : DatabaseTest(), RunsAsUser {
     val moduleEventId = insertEvent(revision = 1, startTime = startTime)
     val notificationEvent = ModuleEventScheduledEvent(moduleEventId, 1)
 
-    assertTrue(notifyTime.isAfter((clock.instant)))
+    assertTrue(notifyTime.isAfter((clock.instant())))
```

Comment:
Oops, I was unclear in my previous comment: this had _too many_ parentheses, not too few. That is, it should be
```kotlin
assertTrue(notifyTime.isAfter(clock.instant))
```
rather than
```kotlin
assertTrue(notifyTime.isAfter((clock.instant)))
```

Reply:
Also, a description of the assertion would be good to differentiate it from the other `assertTrue` in this test.